app/services/hyrax/collection_types/create_service.rb

```diff
@@ -37,28 +68,39 @@ class CreateService
       # @option options [True | False] :nestable if true, collections of this type can be nested
       # @option options [True | False] :discoverable if true, collections of this type can be marked Public and found in search results
       # @option options [True | False] :sharable if true, collections of this type can have participants added for :manage, :deposit, or :view access
+      # @option options [True | False] :share_applies_to_new_works if true, share participant permissions are applied to new works created in the collection
```

Comment:
`[Boolean]` is the way to write this in YARD documentation markup. See http://www.rubydoc.info/gems/yard/file/docs/Tags.md#Class_or_Module_Types